crates/sui-indexer/src/indexer.rs

```diff
@@ -101,9 +103,6 @@ impl Indexer {
         )
         .await?;
 
-        let epochs_to_keep = std::env::var("EPOCHS_TO_KEEP")
```

Comment:
we will still need to keep the env reading, maybe in `start_writer`? b/c this is enabled in production like https://github.com/MystenLabs/sui-operations/blob/main/pulumi/gcp/sui-indexer/Pulumi.mainnet.yaml#L42
it's doable but more changes are needed to entirely get rid of EPOCHS_TO_KEEP as an env var, maybe making it part of indexer config.

Reply:
ah oops, that's a good call

Reply:
added it back for now